net-mail/vpopmail/vpopmail-5.4.33-r5.ebuild

```diff
@@ -0,0 +1,247 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools eutils fixheadtails qmail
+
+HOMEPAGE="http://www.inter7.com/index.php?page=vpopmail"
+DESCRIPTION="Collection of programs to manage virtual email on Qmail servers"
+SRC_URI="mirror://sourceforge/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~hppa ~ia64 ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86"
+IUSE="clearpasswd ipalias maildrop mysql postgres spamassassin"
+REQUIRED_USE="mysql? ( !postgres )"
+
+DEPEND="
+	acct-group/vpopmail
+	virtual/qmail
+	maildrop? ( mail-filter/maildrop )
+	mysql? ( dev-db/mysql-connector-c:0= )
+	postgres? ( dev-db/postgresql:=[server] )
+	spamassassin? ( mail-filter/spamassassin )"
+RDEPEND="${DEPEND}"
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-5.4.9-access.violation.patch
+	"${FILESDIR}"/${PN}-lazy.patch
+	"${FILESDIR}"/${PN}-vpgsql.patch
+	"${FILESDIR}"/${PN}-double-free.patch
+	"${FILESDIR}"/${PN}-5.4.33-vdelivermail-add-static.patch
+	"${FILESDIR}"/${PN}-5.4.33-fix-those-vfork-instances-that-do-more-than-exec.patch
+	"${FILESDIR}"/${PN}-5.4.33-remove-unneeded-forward-declaration.patch
+	"${FILESDIR}"/${PN}-5.4.33-clean-up-calling-maildrop.patch
+	"${FILESDIR}"/${PN}-5.4.33-fix-S-tag-in-case-spamassassin-changed-the-file-size.patch
+	"${FILESDIR}"/${PN}-5.4.33-strncat.patch
+	"${FILESDIR}"/${PN}-5.4.33-unistd.patch
+	"${FILESDIR}"/${PN}-5.4.33-check-crypt-return-value-for-NULL.patch
+	"${FILESDIR}"/${PN}-5.4.33-use-proper-printf-format-strings.patch
+)
+DOCS=(
+	ChangeLog
+	doc/.
+)
+HTML_DOCS=(
+	doc_html/.
+	man_html/.
+)
+
+VPOP_HOME="/var/vpopmail"
+
+pkg_setup() {
+	upgradewarning
+}
+
+src_prepare() {
+	default
+
+	echo 'install-recursive: install-exec-am' \
+		>>"${S}"/Makefile.am || die
+
+	# fix maildir paths
+	sed -i -e 's|Maildir|.maildir|g' \
+		vchkpw.c vconvert.c vdelivermail.c \
+		vpopbull.c vpopmail.c vqmaillocal.c \
+		vuserinfo.c maildirquota.c || die
+
+	# remove vpopmail advertisement
+	sed -i -e '/printf.*vpopmail/s:vpopmail (:(:' \
+		vdelivermail.c vpopbull.c vqmaillocal.c || die
+
+	# automake/autoconf
+	mv "${S}"/configure.{in,ac} || die
+	sed -i -e 's,AM_CONFIG_HEADER,AC_CONFIG_HEADERS,g' \
+		configure.ac || die
+
+	# _FORTIFY_SOURCE
+	sed -i \
+		-e 's/\(snprintf(\s*\(LI->[a-zA-Z_]\+\),\s*\)[a-zA-Z_]\+,/\1 sizeof(\2),/' \
+		vlistlib.c || die
+
+	eautoreconf
+	ht_fix_file cdb/Makefile
+}
+
+src_configure() {
+	local authopts
+	if use mysql; then
+		incdir=$(mysql_config --variable=pkgincludedir || die)
+		libdir=$(mysql_config --variable=pkglibdir || die)
+		authopts+=" --enable-auth-module=mysql"
```

Comment:
Why not make it into an array? This should solve problems with paths containing spaces.

Reply:
should work now